src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -1,42 +1,197 @@
+
+(defn password-form
+  [{:keys [scroll-to-end-fn]}]
+  (reagent/with-let [state               (reagent/atom {:password            ""
+                                                        :repeat-password     ""
+                                                        :accepts-disclaimer? false
+                                                        :focused-input       nil})
+                     password            (reagent/cursor state [:password])
+                     repeat-password     (reagent/cursor state [:repeat-password])
+                     accepts-discalimer? (reagent/cursor state [:accepts-disclaimer?])
+                     focused-input       (reagent/cursor state [:focused-input])]
+    (let [{:keys [long-enough?]
+           :as   validations} (password-validations @password)
+          password-strength   (calc-password-strength validations)
+          empty-password?     (empty? @password)
+          same-passwords?     (= @password @repeat-password)
+          same-length?        (and (not empty-password?)
+                                   (= (count @password) (count @repeat-password)))
+          meet-requirements?  (and (not empty-password?)
+                                   (= password-strength 5)
+                                   same-passwords?
+                                   @accepts-discalimer?)]
+      [:<>
+       [rn/view {:style style/top-part}
+        [header]
+        [password-inputs
+         {:password-long-enough?     long-enough?
+          :passwords-match?          same-passwords?
+          :empty-password?           empty-password?
+          :same-password-lenght?     same-length?
+          :on-input-focus            (fn [input-id]
+                                       (scroll-to-end-fn)
+                                       (reset! focused-input input-id))
+          :on-change-password        #(reset! password %)
+          :on-change-repeat-password #(reset! repeat-password %)}]]
+
+       [rn/view {:style style/bottom-part}
+        (when (= @focused-input :password)
+          [help
+           {:validations       validations
+            :password-strength password-strength}])
+
+        (when (= @focused-input :repeat-password)
+          [rn/view {:style style/disclaimer-container}
+           [quo/disclaimer
+            {:on-change #(reset! accepts-discalimer? %)
+             :checked?  @accepts-discalimer?}
+            (i18n/label :t/password-creation-disclaimer)]])
+
+        [rn/view {:style style/button-container}
+         [quo/button
+          {:disabled (not meet-requirements?)
+           :on-press #(rf/dispatch [:navigate-to :enable-biometrics])}
+          (i18n/label :t/password-creation-confirm)]]]])))
 
 (defn create-password
   []
-  [rn/view {:style {:flex 1}}
-   [background/view true]
-   [page]])
+  (reagent/with-let [scroll-view-ref  (atom nil)
+                     scroll-to-end-fn #(js/setTimeout (.-scrollToEnd @scroll-view-ref)
```

Comment:
When using a small screen device, the UI can't be fully shown :( 
So I decided to add a scroll-view component and  scroll down when an input is focused, please try on a small screen device, IMO the result is good